src/Sylius/Behat/Context/Api/Admin/ManagingPromotionsContext.php

```diff
@@ -255,7 +270,7 @@ public function iEditPromotionToHaveDiscount(PromotionInterface $promotion, floa
     /**
      * @When /^I specify that on ("[^"]+" channel) this action should be applied to items with price greater than "(?:€|£|\$)([^"]+)"$/
      */
-    public function iAddAMinPriceFilterRangeForChannel(ChannelInterface $channel, int $minimum): void
+    public function iAddAMinPriceFilterRangeForChannel(ChannelInterface $channel, int|string $minimum): void
```

Comment:
Why do we need to allow passing `string` here 🤨?

Reply:
There are scenarios that test the validation of promotions, and one of them inputs a string here
[scenario](https://github.com/Sylius/Sylius/blob/1.13/features/promotion/managing_promotions/promotions_filter_validation.feature#L17)